designer/client/outputs/notify-edit.tsx

```diff
@@ -50,6 +50,10 @@ class NotifyEdit extends Component<Props, State> {
         name: condition.name,
         display: condition.displayName,
       })),
+      ...lists.map((list) => ({
+        name: list.name,
+        display: list.title,
```

Comment:
I think maybe we do something like 
```
display: `${list.title} (List)`
```
since this dropdown can get pretty long/confusing.